components/gcp-click-to-deploy/src/DeployForm.tsx

```diff
@@ -482,6 +483,29 @@ export default class DeployForm extends React.Component<any, DeployFormState> {
     }, 10000);
   }
 
+  private _redirectToKFDashboard(dashboardUri: string) {
+    // relying on JupyterHub logo image to be available when the site is ready.
```

Comment:
Is this because of the cors issue? Might be worth elaborating on that per the PR description.

Reply:
done.